.gitignore

```diff
@@ -22,6 +22,13 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+# IDE files
+.idea/
+build
+out/
+*.iml
+*.xml
+=======
```

Comment:
*.iml and out/ already exist in .gitignore, are you sure that it's necessary to add all of these to the file?